common/lib/capa/capa/xqueue_interface.py

```diff
@@ -134,12 +136,17 @@ def _send_to_queue(self, header, body, files_to_upload):
 
     def _http_post(self, url, data, files=None):
         try:
-            r = self.session.post(url, data=data, files=files)
+            response = self.session.post(url, data=data, files=files,
+                                         timeout=(CONNECT_TIMEOUT, READ_TIMEOUT))
```

Comment:
Nit: Please break up the long line using one of the recommended formats from http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html#breaking-long-lines